soh/soh/Enhancements/mods.cpp

```diff
@@ -140,36 +155,82 @@ void RegisterFreezeTime() {
     });
 }
 
-/// Switches Link's age and respawns him at the last entrance he entered.
+/// Switches Link's age and respawns him at the last entrance and position he entered.
 void RegisterSwitchAge() {
```

Comment:
we should probably just register time travel separately, it seems combining the functionality here is making things messy

Reply:
basically instead of setting a the switchage cvar in ocarinatimetravel and adding extra logic here, moving some of the logic in here out to a place that it can be called by switchage and by ocarinatimetravel